src/modules/dashboard/components/DecisionPreview/DecisionPreview.tsx

```diff
@@ -36,11 +38,12 @@ const MSG = defineMessages({
 const decisionData = {
   title: 'Should we build a Discord Bot?',
   description: `I think we should build a Discord bot that integrates with the Dapp and provides our community with greater transperency and also provides more convienience for us to be notified of things happening in our Colony.`,
+  /* Using an HTML string for the dialog content ensures the isDirty prop works as expected */
+  htmlDescription: `<p>I think we should build a Discord bot that integrates with the Dapp and provides our community with greater transperency and also provides more convienience for us to be notified of things happening in our Colony.</p>`,
```

Comment:
Should this be HTML? One of the reasons for us picking TipTap was because it had a JSON input and output format. See here - https://tiptap.dev/guide/output

Reply:
@arrenv I'm only using an HTML string to keep track of the form state, so I can validate that the form is not empty. My cursory attempts just now to do that with an object were ineffective (presumably because two distinct objects with the same shape aren't equal to one another). 

The output format can still be JSON. Is that what you were querying?

Reply:
All good, yeah, that is what I was referring too.